metadata/schema_test.go

```diff
@@ -0,0 +1,26 @@
+package metadata
+
+import (
+	"github.com/featureform/scheduling"
+	"testing"
+)
+
+func TestExecutorTaskRunLockPath(t *testing.T) {
+	type args struct {
+		id scheduling.TaskRunID
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"Zero", args{scheduling.NewTaskRunIdFromString("0")}, "/runlock/0"},
```

Comment:
`NewTaskRunIdFromString` is not handling the error case.

If I pass in a non-numerical arg like "ASDF" it will resolve to an ID of 0, and this test passes. I believe it should fail with `NewInternalError`